courses-app/src/App.jsx

```diff
@@ -1,7 +1,27 @@
+import { useState } from 'react';
+import Header from './components/Header/Header';
+import Courses from './components/Courses/Courses';
+import CreateCourse from './components/CreateCourse/CreateCourse';
+
 import './App.css';
 
 function App() {
-	return <div className='App'>START</div>;
+	const [isCreateCourse, setIsCreateCourse] = useState(false);
```

Comment:
Better to change a naming for better representation of what the constant is responsible for.
_isCreateCourseRoute_ gives more meaning.